model/config.go

```diff
@@ -88,18 +88,17 @@ const (
 
 	SITENAME_MAX_LENGTH = 30
 
-	SERVICE_SETTINGS_DEFAULT_SITE_URL           = "http://localhost:8065"
-	SERVICE_SETTINGS_DEFAULT_TLS_CERT_FILE      = ""
-	SERVICE_SETTINGS_DEFAULT_TLS_KEY_FILE       = ""
-	SERVICE_SETTINGS_DEFAULT_READ_TIMEOUT       = 300
-	SERVICE_SETTINGS_DEFAULT_WRITE_TIMEOUT      = 300
-	SERVICE_SETTINGS_DEFAULT_IDLE_TIMEOUT       = 60
-	SERVICE_SETTINGS_DEFAULT_MAX_LOGIN_ATTEMPTS = 10
-	SERVICE_SETTINGS_DEFAULT_ALLOW_CORS_FROM    = ""
-	SERVICE_SETTINGS_DEFAULT_LISTEN_AND_ADDRESS = ":8065"
-	SERVICE_SETTINGS_DEFAULT_GFYCAT_API_KEY     = "2_KtH_W5"
-	SERVICE_SETTINGS_DEFAULT_GFYCAT_API_SECRET  = "3wLVZPiswc3DnaiaFoLkDvB4X0IV6CpMkj4tf2inJRsBY6-FnkT08zGmppWFgeof"
-
+	SERVICE_SETTINGS_DEFAULT_SITE_URL              = "http://localhost:8065"
```

Comment:
probably gofmt would reduce the number of spaces at the end and probably remove this whole bulk of changes.